File: app/src/main/java/za/org/grassroot2/services/account/AuthConstants.java
package za.org.grassroot2.services.account;

public interface AuthConstants {
    String ACCOUNT_TYPE = "za.org.grassroot2";
    String ACCOUNT_NAME = "Grassroot";
    String AUTH_TOKENTYPE = ACCOUNT_TYPE;

    // for calls to auth server
    String AUTH_CLIENT_TYPE        = "ANDROID";
    String USER_DATA_CURRENT_TOKEN = "current_token";
    String USER_DATA_LOGGED_IN     = "logged_in";
}

File: app/src/main/java/za/org/grassroot2/services/account/AuthConstants.kt
package za.org.grassroot2.services.account

interface AuthConstants {
    companion object {
        const val ACCOUNT_TYPE = "za.org.grassroot2"
        const val ACCOUNT_NAME = "Grassroot"
        const val AUTH_TOKENTYPE = ACCOUNT_TYPE

        // for calls to auth server
        const val AUTH_CLIENT_TYPE = "ANDROID"
        const val USER_DATA_CURRENT_TOKEN = "current_token"
        const val USER_DATA_LOGGED_IN = "logged_in"
    }
}
